pyomo/contrib/fbbt/interval.py

```diff
@@ -282,7 +282,15 @@ def _inverse_power2(zl, zu, xl, xu, feasiblity_tol):
 
 
 def exp(xl, xu):
-    return math.exp(xl), math.exp(xu)
+    if xl >= 100:
+        lb = math.inf
+    else:
+        lb = math.exp(xl)
+    if xu >= 100:
```

Comment:
Where did 100 come from?  I would prefer to see the constant pulled from Python or with a try-except block.  Is there a similar issue for the lower limits and underflow?

Reply:
@jsiirola Good suggestion. I went with the try-except approach. There is not a similar issue for the lower limits and underflow. If the bounds are too small (large negative numbers), then the ```exp``` function from ```math``` will just return 0.